packages/v4/src/content/design-guidelines/components/clipboard-copy/clipboard-copy.md

```diff
@@ -38,6 +37,15 @@ An expandable clipboard copy allows for long lines of text to be stored in an ex
 
 <img src="./img/expandable.png" alt="Example of an expandable clipboard copy component" width="455"/>
 
+**Inline**
+
+An inline clipboard copy is placed directly inline with other text or used on it's own when you want a lighter weight clipboard copy treatment.
+
+<img src="./img/inline-one-action.png" alt="Example of an inline clipboard copy component with one action" width="355"/>
```

Comment:
Last comment is that these images are a little blurry! Make sure to export the art board at x2 PNG, but keep the width set to the original sketch art board width!

Reply:
I can do that, thank you for letting me know. I'll re-export the images.